### ISSUE TITLE
3 dashes ligature as solid line

### ISSUE THREAD

**Issue by maudrid** [tonsky/FiraCode#1552]
Hi. I think I was using an older version of the font before.
I got a new computer (Linux) and installed FiraCode.
Now when I do 3 dashes "---", I can see that it converts it to a ligature, but it has a small space separating the dashes.
On my other computer it creates a solid line. I like the solid line better for separating YAML blocks.
Is there a way to return the previous behavior with the latest version of the font?
In VSCode I have tried setting `"editor.fontLigatures"` to `'calt'`, but that did not work.
Any advice would be appreciated.

**Comment by tonsky** [comment]
No, it’s an unfortunate artefact of new implementation of “infinite arrows”. Maybe `--` and `---` deserve a hand-crafted ligature though
